From 6672203183b1d3d992e19f47a43cb89b7d798fd0 Mon Sep 17 00:00:00 2001
From: Logan Chien <loganchien@google.com>
Date: Mon, 27 Aug 2012 12:54:37 +0800
Subject: [PATCH 2/6] Rename ANDROIDEABI to ANDROID.

The code releted to ANDROIDEABI is not specific to
ARM EABI.  We need them in Intel and MIPS backend
as well.
---
 include/llvm/ADT/Triple.h |    2 +-
 lib/Support/Triple.cpp    |    5 +++--
 2 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/include/llvm/ADT/Triple.h b/include/llvm/ADT/Triple.h
index f5f99d0..5e03034 100644
--- a/include/llvm/ADT/Triple.h
+++ b/include/llvm/ADT/Triple.h
@@ -108,7 +108,7 @@ public:
     GNUEABIHF,
     EABI,
     MachO,
-    ANDROIDEABI
+    ANDROID
   };
 
 private:
diff --git a/lib/Support/Triple.cpp b/lib/Support/Triple.cpp
index 44a1b38..cabacfa 100644
--- a/lib/Support/Triple.cpp
+++ b/lib/Support/Triple.cpp
@@ -132,7 +132,7 @@ const char *Triple::getEnvironmentTypeName(EnvironmentType Kind) {
   case GNUEABI: return "gnueabi";
   case EABI: return "eabi";
   case MachO: return "macho";
-  case ANDROIDEABI: return "androideabi";
+  case ANDROID: return "android";
   }
 
   llvm_unreachable("Invalid EnvironmentType!");
@@ -298,7 +298,8 @@ static Triple::EnvironmentType parseEnvironment(StringRef EnvironmentName) {
     .StartsWith("gnueabi", Triple::GNUEABI)
     .StartsWith("gnu", Triple::GNU)
     .StartsWith("macho", Triple::MachO)
-    .StartsWith("androideabi", Triple::ANDROIDEABI)
+    .StartsWith("android", Triple::ANDROID)
+    .StartsWith("androideabi", Triple::ANDROID)
     .Default(Triple::UnknownEnvironment);
 }
 
-- 
1.7.7.3

